insurance-svg-template.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
Does insurance and mosipid support datamodel 2.0 to support these templates ?

Reply:
Since insurance and mosipid do not support datamodel 2.0, hence removing the templates. It should be updated for mock-plugin as MockCSVDataProviderPlugin plugin is a part of mock-certify-plugin which supports data model 2.0